functions_and_scope_lab.js

```diff
@@ -0,0 +1,286 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+const largestOfTwo = (num1, num2) =>{
+  if(num1 > num2){
+    return num1;
+  }else if(num1 === num2){
```

Comment:
If they are equal, return the second one <==== this is what the question asked for.